docs/terminology.md

```diff
@@ -0,0 +1,47 @@
+# Terminology for control plane
+Contains the definitions shared across design documents around prototyping a kube-like control plane (in KCP).  This is a derivative work of other design documents intended to frame terminology.  All future statements that may be changed by designs is covered by those designs, and not duplicated here.
+
+## Logical cluster
+A logical cluster is a way to subdivide a single kube-apiserver + etcd storage into multiple clusters (different APIs, separate semantics for access, policy, and control) without requiring multiple instances.  A logical cluster is a mechanism for achieving separation, but may be modelled differently in different use cases.  A logical cluster is similar to a virtual cluster as defined by sig-multicluster, but is able to amortize the cost of a new cluster to be near-zero memory and storage so that we can create tens of millions of empty clusters cheaply.
```

Comment:
Technically as implemented today there is actually zero cost to a "new empty" logical cluster, since there is no book-keeping...

Reply:
changed to `zero or near zero`. Let me know what do you think.